.idea/gradle.xml

```diff
@@ -1,3 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project version="4" />
-
+<project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
+  <component name="GradleSettings">
+    <option name="linkedExternalProjectsSettings">
+      <GradleProjectSettings>
+        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="modules">
+          <set>
+            <option value="$PROJECT_DIR$" />
+            <option value="$PROJECT_DIR$/apps" />
+            <option value="$PROJECT_DIR$/buildSrc" />
+            <option value="$PROJECT_DIR$/modules/javafx.base" />
+            <option value="$PROJECT_DIR$/modules/javafx.controls" />
+            <option value="$PROJECT_DIR$/modules/javafx.fxml" />
+            <option value="$PROJECT_DIR$/modules/javafx.graphics" />
+            <option value="$PROJECT_DIR$/modules/javafx.media" />
+            <option value="$PROJECT_DIR$/modules/javafx.swing" />
+            <option value="$PROJECT_DIR$/modules/javafx.swt" />
+            <option value="$PROJECT_DIR$/modules/javafx.web" />
+            <option value="$PROJECT_DIR$/tests/system" />
+          </set>
+        </option>
+      </GradleProjectSettings>
+    </option>
+  </component>
+</project>
```

Comment:
I can see the same modules in project structure with or without this change. Is this required for build to work ?

Reply:
This enabled the IntelliJ gradle plugin for me, so e.g. the support for doing the tasks in the ide. And with it, I was able to run normal unit tests from the ide (see also screenshot(s) below)
![image](https://user-images.githubusercontent.com/66004280/120438497-c0562280-c381-11eb-8bda-f1d17d8b30ac.png)